sentry-samples/sentry-samples-android/src/main/java/io/sentry/samples/android/SampleInnerFragment.kt

```diff
@@ -0,0 +1,28 @@
+package io.sentry.samples.android
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import io.sentry.Sentry
+import io.sentry.samples.android.databinding.FragmentSampleInnerBinding
+
+class SampleInnerFragment : Fragment() {
```

Comment:
This inner fragment is made to show that registered `FragmentLifecycleCallbacks` is indeed recursive.